src/app/pages/swap/components/swap-assets-pair/swap-assets-pair.tsx

```diff
@@ -0,0 +1,25 @@
+import { FiArrowDown } from 'react-icons/fi';
+
+import { Box } from '@stacks/ui';
+
+import { SwapAsset } from '../../hooks/use-swap';
+import { SwapAssetItem } from './swap-asset-item';
+import { SwapAssetsPairLayout } from './swap-assets-pair.layout';
+
+interface SwapAssetsPairProps {
+  amountFrom: string;
+  amountTo: string;
```

Comment:
confused what these mean? The amount is a range?

Reply:
No, there is an amount being swapped `from` an asset `to` another asset. The `amountTo` is calculated on the exchange rate and set based on the `amountFrom` entered by the user.